### DISCUSSION TITLE
remind microphone muted sound

### DISCUSSION THREAD

**Discussion by alexei-petru** [SaifAqqad/AHK_MicMute#46] [Ideas]
Hey, great app. Will be awesome if you can add a sound reminder when you speak and your microphone is muted, 
if you can go further and made this sound to be configurable and can switch to simple muted in tray or even add a icon to show muted-sound.
Can you do this ?

**Comment by SaifAqqad** [comment]
> sound reminder when you speak and your microphone is muted

That's not possible because MicMute mutes the microphone at the OS level (It's muted for all applications)

It would need to have the actual microphone unmuted to be able to detect sounds when you speak, and somehow still be muted for all other applications
